layouts/shortcodes/finding-vulnerability-logic.md

```diff
@@ -0,0 +1,6 @@
+When you select [Full Report + Finding Details](/getting-started/what-to-expect/report-contents/)
```

Comment:
With this file, I try to describe our doll, where 

- Each finding specifies a vulnerability type
- Each finding (as a whole) is also a vulnerability report

I'm open to other ways to describe what we do -- or I can leave it out completely (and keep it "mysterious" ...)

Reply:
I think the way I would describe this is: 
When you select 'full report + finding details', the application displays the same contents as the Full Report, and an appendix containing each of the individual finding reports from the Findings tab, listed in order of discovery.

If that's clearer?